src/components/filters/MeasureValueFilter/DropdownAfmWrapper.tsx

```diff
@@ -12,14 +12,16 @@ import {
 } from "../../../interfaces/MeasureValueFilter";
 
 export interface IDropdownProps {
-    filter: VisualizationInput.IMeasureValueFilter;
+    filter?: VisualizationInput.IMeasureValueFilter;
```

Comment:
Why optional now?

Reply:
The filter must be required, otherwise, you could end up with no knowing the measure the filter is linked to and that should never happen.